django_freeradius/migrations/0018_add_default_profile.py

```diff
@@ -0,0 +1,41 @@
+import swapper
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.db import migrations
+
+def add_default_profiles(apps, schema_editor):
+    RadiusProfile = swapper.load_model('django_freeradius', 'RadiusProfile')
+    default_profile = RadiusProfile.objects.filter(default=True)
+    if not default_profile.exists():
+        limited_user = RadiusProfile(name="Limited User", default=True)
```

Comment:
excuse me, is this profile doing anything at all? Where are the limits defined here?

Reply:
Nothing as of now. Wanted to confirm the limits from you.

Reply:
The parameters are what I wanted to confirm? What limits do we put on each of them